.github/workflows/continuous-integration.yml

```diff
@@ -421,30 +421,23 @@ jobs:
         uses: actions/cache@69d9d449aced6a2ede0bc19182fadc3a0a42d2b0 # v3
         with:
           path: .cache
-          key: docker-compose-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('**/composer.lock') }}
+          key: docker-compose-${{ hashFiles('frontend/package-lock.json', 'print/package-lock.json', 'api/composer.lock') }}
           restore-keys: |
             docker-compose-
 
       # start necessary containers
       - run: docker compose up -d php caddy frontend print browserless database docker-host
-
-      # pull cypress while container are starting up
-      - run: docker pull cypress/included:12.7.0
-
-      # wait for backend & frontend to be up and running
-      - run: bash wait-for-container-startup.sh
-        timeout-minutes: 5
-
-      # run end-to-end tests
-      - run: docker run -v $PWD:/e2e -w /e2e --network host cypress/included:12.7.0
-        working-directory: frontend
+      
+      - uses: cypress-io/github-action@v5
+        with:
+          working-directory: e2e
+          browser: chrome
```

Comment:
It could make sense to test in more browsers than just chrome. Do you see an easy way to do this? If not, we can leave it like this and maybe set up a matrix build later

Reply:
The cypress action supports chrome, edge and firefox. So yes, with a matrix job we could test multiple. We'd sacrifice on test timing, though.

Reply:
Matrix jobs run in parallel, so time shouldn't be that much of an issue

Reply:
#3320